manual/sphinx/conf.py

```diff
@@ -36,21 +37,34 @@
 on_readthedocs = os.environ.get("READTHEDOCS") == "True"
 
 if on_readthedocs:
-    from unittest.mock import MagicMock
-
-    class Mock(MagicMock):
-        __all__ = ["foo",]
-        @classmethod
-        def __getattr__(cls, name):
-            return MagicMock()
-
-    MOCK_MODULES = [
-        'boutcore',
-        'bunch',
-        'h5py',
-        'netCDF4',
-    ]
-    sys.modules.update((mod_name, Mock()) for mod_name in MOCK_MODULES)
+    print(os.environ)
+    print(sys.argv)
+    python = sys.argv[0]
+    pydir = "/".join(python.split("/")[:-2])
+    os.system("which clang-format")
+    os.system("which clang-format-6.0")
+    os.system(
+        "git clone https://github.com/mpark/variant.git ../../externalpackages/mpark.variant"
```

Comment:
I'm guessing this means RTD doesn't clone the top-level repo so we can't use the submodules?

Reply:
Yes, I forgot the exact failure but git complained it wouldnt know the url for cloning.